app/+html.tsx

```
@@ -0,0 +1,44 @@
+                    content="width=device-width, initial-scale=1, shrink-to-fit=no"
+                />
+
+                {/*
+          Disable body scrolling on web. This makes ScrollView components work closer to how they do on native.
+          However, body scrolling is often nice to have for mobile web. If you want to enable it, remove this line.
+        */}
+                <ScrollViewStyleReset />
+
+                {/* Using raw CSS styles as an escape-hatch to ensure the background color never flickers in dark-mode. */}
+                <style
+                    dangerouslySetInnerHTML={{ __html: responsiveBackground }}
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)